src/assets.ts

```diff
@@ -382,43 +354,10 @@ export function createBlazorWebAssemblyDevServerLaunchConfiguration(workingDirec
     "type": "coreclr",
     "request": "launch",
     "program": "dotnet",
-    "args": ["run", "--no-build"],
+    "args": ["run"],
```

Comment:
That's a good change. I was a bit concerned about people getting tripped up by `--no-build` before.